keras/engine/compile_utils.py

```diff
@@ -107,7 +107,7 @@ def __init__(self, losses, loss_weights=None, output_names=None):
     self._user_losses = losses
     self._user_loss_weights = loss_weights
 
-    self._losses = losses
+    self._losses = copy.copy(losses)
```

Comment:
As far as I can tell (cf. the `compile()` method) the `losses` argument is *not* guaranteed to be a list/dict and thus is not safe to copy.